### ISSUE TITLE
Building TRT Engine on Windows 11 Results in LoraConfig.from_hf() Arguments Error

### ISSUE THREAD

**Issue by geeeethub** [NVIDIA/trt-llm-rag-windows#12]
I currently run an i9 with a RTX 3080. I was trying to build a custom engine using the instructions.

I download a copy of meta-llama/Llama-2-13b-chat-hf model and the LLaMa 2 13B AWQ int4 checkpoints model.pt file, when I run the command below (I double checked the dirs and even used direct notation, i.e., C:\users\[username]\...):

`python .\build.py --model_dir %USERPROFILE%\inference\models\Llama-2-13b-chat-hf\ --quant_ckpt_path %USERPROFILE%\inference\models\ --dtype float16 --use_gpt_attention_plugin float16 --remove_input_padding --use_gemm_plugin float16 --use_weight_only --weight_only_precision int4_awq --per_group --enable_context_fmha --max_batch_size 1 --max_input_len 3000 --max_output_len 1024 --output_dir %USERPROFILE%\inference\models\engines\`

But it always results in:

```
Traceback (most recent call last):
  File "C:\Users\MYNAME\inference\TensorRT-LLM\examples\llama\build.py", line 1040, in <module>
    args = parse_arguments()
  File "C:\Users\MYNAME\inference\TensorRT-LLM\examples\llama\build.py", line 627, in parse_arguments
    lora_config = LoraConfig.from_hf(args.hf_lora_dir,
TypeError: LoraConfig.from_hf() takes 3 positional arguments but 4 were given
```

I uninstalled all libraries, CUDA, etc. and reinstalled using the instructions... redownloaded the models and double checked versions. It still gives the same error.

Any thoughts?

**Comment by sam-india-007** [comment]
I am facing the same issue currently, please look into it.
I am building the Llama example given in the repo

Edit: It seems I am in the wrong repo, I was working with the TensorRT-LLM repo but it resulted in this same error

**Comment by sarashs** [comment]
Same problem. I found the source of the error:

line 37 of lora_manager.py:

    @classmethod
    def from_hf(cls, hf_lora_dir, hf_modules_to_trtllm_modules):
    
 so it gets two arguments plus cls. However, where it is instantiated they have:
 
     lora_config = LoraConfig.from_hf(args.hf_lora_dir,
                                     hf_modules_to_trtllm_modules,
                                     trtllm_modules_to_hf_modules)
which has 3 arguments plus cls (total of 4) with trtllm_modules_to_hf_modules being redundant.
I fixed that and it lead to another error which in turn lead to another error...

**Comment by Hjhirp** [comment]
I have solved it. The problem lies in the file path. When running the code, the model locates the Python libs in my user, and when installing, I install it in my program files. Thus, the code in my user lib had an older version of the tensorrtllm that I had installed previously. A naive quick-fix I applied was to copy and paste the tensorrtllm dir to that user lib manually from my git cloned dir. The proper fix would be to install everything again and set paths correctly.

Hope this helps out!
